src/index.html

```diff
@@ -1,13 +1,330 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="icon"
+      type="image/vnd.microsoft.icon"
+      href="//static.bose.com/etc/designs/bose/consumer-products-2016/favicon.ico"
+    >
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="header__upper">
+        <a href="/layout_miami" class="header__logo">
+          <img
+            src="./images/logo.png"
+            alt="Bose"
+            class="header__img"
+          >
+        </a>
+
+        <div class="header__icon">
+          <button class="header__phone">
+            <a href="tel:+1 234 5555-55-55" class="header__link"></a>
+          </button>
+
+          <button class="header__toggler"></button>
+        </div>
+      </div>
+
+
+      <nav class="nav header__nav">
+        <div class="nav__upper">
+          <a href="/layout_miami" class="nav__logo">
+            <img
+              src="./images/logo.png"
+              alt="Bose"
+              class="nav__img"
+            >
+          </a>
+
+          <div class="nav__icon">
+            <button class="nav__toggler"></button>
+          </div>
+        </div>
+
+        <div class="nav__menu">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="/layout_miami" class="nav__link">home</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#about-us" class="nav__link">about us</a>
```

Comment:
These links also should send the user to the respective sections. I see that the only home link is functional at the moment. Should be a relatively easy fix